### PR TITLE
Release 3.0.0 beta3

### PR DESCRIPTION
**Pull request by no-reply**
Prepare beta3 release

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: template.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Hack for https://github.com/rails/rails/issues/35153
 gsub_file 'Gemfile', /^gem ["']sqlite3["']$/, 'gem "sqlite3", "~> 1.3.0"'
-gem 'hyrax', '3.0.0-beta2'
+gem 'hyrax', '3.0.0-beta3', path: '~/src/hyrax'
```
Comment (cjcolvar) [inline template.rb]: I think you should remove the path part here.  Probably vestigial from testing.

File: README.md [hunk 2]
```diff
@@ -143,11 +139,11 @@ Starting up Redis will depend on your operating system, and may in fact already
 
 ## Rails
 
-Hyrax requires Rails 5. We recommend the latest Rails 5.1 release.
+Hyrax requires Rails 5. We recommend the latest Rails 5.2 release.
 
 ```
 # If you don't already have Rails at your disposal...
-gem install rails -v 5.1.7
+gem install rails -v 5.2.2
```
Comment (cjcolvar) [inline README.md]: Why not the latest 5.2.x rails (5.2.3)?

File: template.rb (hunk 1 above)
Comment (no-reply) [reply template.rb]: thanks! good catch

File: README.md (hunk 2 above)
Comment (no-reply) [reply README.md]: :+1:

File: README.md [hunk 3]
```diff
@@ -163,7 +159,7 @@ NOTE: The steps need to be done in order to create a new Hyrax based app.
 Generate a new Rails application using the template.
 
 ```
-rails _5.1.7_ new my_app -m https://raw.githubusercontent.com/samvera/hyrax/v3.0.0.pre.beta2/template.rb
+rails _5.2.2_ new my_app -m https://raw.githubusercontent.com/samvera/hyrax/v3.0.0.pre.beta3/template.rb
```
Comment (cjcolvar) [inline README.md]: Missed this one:
```suggestion
rails _5.2.3_ new my_app -m https://raw.githubusercontent.com/samvera/hyrax/v3.0.0.pre.beta3/template.rb
```

**Pull request merged**